nexus_constructor/stream_fields_widget.py

```diff
@@ -345,20 +352,14 @@ def _record_advanced_f142_values(self, stream: F142Stream):
         Save the advanced f142 properties to the stream data object.
         :param stream: The stream data object to be modified.
         """
-        stream.nexus_indices_index_every_mb = self.f142_index_every_mb_spinner.value()
-        stream.nexus_indices_index_every_kb = self.f142_index_every_kb_spinner.value()
-        stream.store_latest_into = self.f142_store_latest_into_spinner.value()
+        raise NotImplementedError
```

Comment:
Remember to remove advanced option from the UI too.

Reply:
Add `cue_interval` and `chunk_size` at appropriate place and also in advanced config in UI.

Reply:
https://jira.esss.lu.se/browse/ECDC-2725 should be addressed in this ticket